### ISSUE TITLE
[Feature Request] Ability to stop and start scanning while using Html5QrcodeScanner

### ISSUE THREAD

**Issue by ZhuanM** [mebjas/html5-qrcode#285]
**Is your feature request related to a problem? Please describe.**
Not having the ability to stop the scanning process (without removing/cleaning the entire scanner itself - but simply not letting the scanner scan) and then being able to start scanning again while using Html5QrcodeScanner makes no sense since it is doable (from what I've read on the documentation) with Html5Qrcode by using the "start" and "stop" functions.

**Describe the solution you'd like**
I'd like to have the ability to stop and start scanning while using Html5QrcodeScanner by having the "start" and "stop" functions that are available with Html5Qrcode.

**Describe alternatives you've considered**
The alternative is to use Html5Qrcode but that means that the developer has to go through rebuilding the entire user interface just to have the "start" and "stop" functions available.

**Comment by mebjas** [comment]
> I'd like to have the ability to stop and start scanning while using Html5QrcodeScanner by having the "start" and "stop" functions that are available with Html5Qrcode.

This makes sense, would you like the whole UI to reset to beginning in that case?

**Comment by ZhuanM** [comment]
> This makes sense, would you like the whole UI to reset to beginning in that case?
>
Well, if it were to reset the UI then it would be the same as using the "clean" function alongside with re-creating a new scanner Object. I think a new functionality that would more useful for this library to have is the ability to not allow further scans but without clearing the entire scanner from the UI. Let's say that the developer wants the scanning to stop after, for example, the first or second or even third scan, but still without closing the scanner and by still showing the video camera feed with the scan box, and after a certain point in the code to be able to have it functionally scanning again.

**Comment by mebjas** [comment]
So it's expected to work like a `stop()` method should programatically trigger the stop button?

**Comment by ZhuanM** [comment]
I want to apologize for misunderstanding the ``stop()`` and ``start()`` functionality. As I worked even more with the Html5Qrcode class I became more familiar with them and it's not what I really intented to request as a feature. What I actually intented as a feature request is in my latest comment - the ability to be able to stop the scanning process with a **NEW** function (which doesn't exist on any class) **without** clearing the UI and while continuing to show the video camera feed with the QR shading scan box. Alongside with that **new** stopping function there should obviously also be a **NEW** function (which also doesn't exist on any class) which would be able to have the scanner be functionally scanning again and being able to read new Barcodes. Basically the whole idea is to have **2 new functions** - stop reading and start reading.

**Comment by mebjas** [comment]
I think what you are looking for is `pause()` and `resume()` methods both at the `Html5QrcodeScanner` and `Html5Qrcode` level.

**Comment by ZhuanM** [comment]
Yes, this would be a more accurate naming of the functions I intented to request.

**Comment by S-Rajkumar** [comment]
> 
> 
> So it's expected to work like a `stop()` method should programatically trigger the stop button?

is this done ?
i need to stop scanning by programatically.

**Comment by svillacreses** [comment]
Hey anybody else having trouble stopping the scanner? (stopping not pausing it)

Please let me know if there is a workaround for stopping it using the Html5QrcodeScanner
